Tests/BFKitTests/Linux/Foundation/ArrayExtensionTests.swift

```diff
@@ -47,6 +49,30 @@ class ArrayExtensionTests: XCTestCase {
         super.tearDown()
     }
     
+    func testCircleObjectAt() {
+        let object = array.circleObject(at: 6)
+        
+        XCTAssert(object == 2)
+    }
+    
+   func testRandom() {
+        let element = array.random()
+        for i in array {
+            if "\(i)" == "\(element)" {
+                XCTAssert("\(i)" == "\(element)")
+            } else {
+                XCTAssert("\(i)" != "\(element)")
+            }
+        }
+        XCTFail()
+    }
+    
+    func testRemove() {
+        var arr = array
```

Comment:
Rename it with something like `arrayCopy` or something else. I'm sorry I don't like short variable names